docs/ossm/ossm2-migration/cert-manager/README.md

```diff
@@ -0,0 +1,301 @@
+NAME                                         CLUSTER        CDS        LDS        EDS        RDS        ECDS     ISTIOD                                   VERSION
+details-v1-9979968fb-776jq.bookinfo          Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+productpage-v1-8669b4d5c8-hshtz.bookinfo     Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+ratings-v1-bbb89988d-tcgvp.bookinfo          Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+reviews-v1-75b6949cf4-7kbdm.bookinfo         Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+reviews-v2-64f68558b-gsxc4.bookinfo          Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+reviews-v3-596954cfd6-jnb6n.bookinfo         Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+```
+ -->
+
+Starting with a `ServiceMeshControlPlane` with cert-manager configured:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  ...
+  security:
+    certificateAuthority:
+      cert-manager:
+        address: cert-manager-istio-csr.cert-manager.svc:443
+      type: cert-manager
+    dataPlane:
+      mtls: true
+    identity:
+      type: ThirdParty
+    manageNetworkPolicy: false
+```
+
+You will need to perform these updates to your istio-csr deployment:
+
+- The `app.istio.revisions` field needs to include your 3.0 control plane revision _before_ you create your `Istio` resource.
+
+  Adding your 3.0 control plane revision to your istio-csr deployment will ensure that proxies can properly communicate with the 3.0 control plane.
+
+  ```console
+  helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+      --install \
+      --reuse-values \
+      --namespace cert-manager \
+      --wait \
+      --set "app.istio.revisions={basic,ossm3-v1-24-1}"
+  ```
+
+  Depending on whether you will use a `RevisionBased` update strategy or an `InPlace` update strategy, your revision name will vary. If using an `InPlace` strategy, your revision name will match your `Istio` name. If using a `RevisionBased` strategy, revision names use the following format, <istio-name>-v<major_version>-<minor_version>-<patch_version>. For example: `ossm3-v1-24-1`.
+
+- The `app.controller.configmapNamespaceSelector` field needs to be either unset _before_ the migration begins or updated _after_ you have completed your migration.
+
+  If you have set the `app.controller.configmapNamespaceSelector` field on your istio-csr deployment to `maistra.io/member-of`, you will need to update this accordingly. If you haven't set this field, you can keep it unset.
```

Comment:
Is it possible to have multiple labels defined in `app.controller.configmapNamespaceSelector`? It would be nice to include also some example when you need to keep `maistra.io/member-of` so 2.x is still working while 3.0 namespace would use a different label.

Reply:
>It would be nice to include also some example when you need to keep maistra.io/member-of so 2.x is still working while 3.0 namespace would use a different label.

This is where you need to update the selector **only after** completing the migration and have also added the new injection label to each namespace.

Reply:
I'll reword it to make it more clear that you don't always need the injection label.